### ISSUE TITLE
Docker image binds port 11885

### ISSUE THREAD

**Issue by adriansmares** [TheThingsNetwork/lorawan-stack#466]
**Summary:** 

The docker image binds port `11885` even though this port is no longer used. This results in the CLI no longer being able to login, since the bind will fail.

**Steps to Reproduce:** 

1. Start the docker container
2. Attempt to `ttn-lw-cli login`

Not necessary.

**What do you see now?**

```
 ERROR Could not listen for OAuth callback. Try re-running this command with --callback=false error=listen tcp :11885: bind: address already in use
listen tcp :11885: bind: address already in use
exit status 255
```

**What do you want to see instead?**

No error.

**Environment:**

Latest master.

**How do you propose to implement this?**

Remove the port `11885` bind from the `Dockerfile`.

**Can you do this yourself and submit a Pull Request?**

Yes.

**Comment by KrishnaIyer** [comment]
The port indeed needs to be removed. But have you tested with the newest CLI? 
https://github.com/TheThingsNetwork/lorawan-stack/pull/433
It uses a local callback and won't redirect to `11885`.

**Comment by adriansmares** [comment]
Closed by #467
